python/lsst/obs/decam/instrument.py

```diff
@@ -0,0 +1,86 @@
+# This file is part of obs_decam.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Butler instrument description for the Dark Energy Camera.
+"""
+
+__all__ = ("DarkEnergyCamera",)
+
+import os
+
+from lsst.afw.cameraGeom import makeCameraFromPath, CameraConfig
+from lsst.daf.butler.instrument import Instrument
+from lsst.obs.decam.decamFilters import DECAM_FILTER_DEFINITIONS
+
+from lsst.utils import getPackageDir
+
+
+class DarkEnergyCamera(Instrument):
+    """Gen3 Butler specialization class for the DECam instrument.
+    """
+    filterDefinitions = DECAM_FILTER_DEFINITIONS
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        packageDir = getPackageDir("obs_decam")
+        self.configPaths = [os.path.join(packageDir, "config")]
+
+    @classmethod
+    def getName(cls):
+        return "DECam"
+
+    def getCamera(self):
+        path = os.path.join(getPackageDir("obs_decam"), "decam", "camGeom")
```

Comment:
I think the long term plan is for these to go into obs_decam_data package.

Reply:
That sounds like a "leave it for (much) later" point.